_projects/trm.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "TRM algorithm"
+show-title: false
+layout: project
+excerpt: "TRM algorithm is a package to compute the Transmission Reliability Margin."
+header:
+  image: /assets/images/trm/logo_trm.png
+  teaser: assets/images/trm/logo_trm.png
+github: https://github.com/rte-france/Transmission-Reliability-Margin-Algorithm
+license: MPL 2.0
+license-url: https://www.mozilla.org/en-US/MPL/2.0/
+documentation:  https://github.com/rte-france/Transmission-Reliability-Margin-Algorithm/blob/main/doc/algorithm.md
+tags: [MPL2.0]
+---
+
+TRM algorithm is a package that computes the Transmission Reliability Margin, aiming to be a translation in Java code of the [article 22 of the CACM regulation 2015/1222](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:32015R1222#d1e2148-24-1).
+The project relies heavily on the [PowSybl environment](https://www.powsybl.org/index.html).
```

Comment:
you can link to the powsybl page of the website, it's better to avoid leaving the website when we can
```suggestion
The project relies heavily on the [PowSybl environment](/projects/powsybl).
```

Reply:
Fixed